### ISSUE TITLE
wrong results

### ISSUE THREAD

**Issue by stas-webdeveloper** [WebDevSimplified/Vanilla-JavaScript-Calculator#2]
0.1 + 0.2  and dividing by zero are giving the incorrect result. I know it is a JS fault, but I was hoping to find a solution here.

**Comment by Psingh17** [comment]
Hi @stepanoffweb , did you get the solution?

**Comment by therealbenpai** [comment]
also, the css doesn't want to follow

**Comment by VorgeRyan** [comment]
um it wont even log the numbers

**Comment by HappyCode2000** [comment]
The JS part doesn't work plzzzz help T^ T

**Comment by pauliesnug** [comment]
It works for me.

**Comment by Prashoon123** [comment]
Yeah, when I am **dividing 0 by 0** I am getting a bug. But overall the Calculator App is **fantastic**.

**Comment by shakib368** [comment]
there is some semicolon problem as well

**Comment by Nikhil27b** [comment]
> Yeah, when I am **dividing 0 by 0** I am getting a bug. But overall the Calculator App is **fantastic**.

#19  Fixed

**Comment by chasekhandelwal** [comment]
Yess I am also I find Problem When I was typing 0/0.. The  app is Good..
